Add YASA Flaskified details to README and FAQ

bartromb opened the pull request:
This pull request adds information about YASA Flaskified, a web-based application built using the YASA library, to the project's documentation:

    README.rst:
        Added a "Related Projects" section with a brief description of YASA Flaskified.
        Included a link to the YASA Flaskified GitHub repository.

    docs/faq.rst:
        Added a collapsible section titled "Related Projects" under "Others."
        Provided details about YASA Flaskified's features, deployment options, and repository link in a structured FAQ format.

These changes aim to introduce users to YASA Flaskified as an extension of YASA for those who prefer a web-based interface or need deployment options for their own servers.

raphaelvallat commented on README.rst:
```diff
@@ -144,3 +144,12 @@ Citation
+Related Projects
+~~~~~~~~~~~~~~~~~
+
+Looking for a web-based solution to utilize YASA? Check out **YASA Flaskified**: a Flask-based application designed to process and analyze EEG data via an intuitive web interface. 
```
I would prefer some kind of bullet list here, e.g.:

```rst
- `YASA Flaskified <https://github.com/bartromb/YASAFlaskified>`_: a Flask-based application designed to process and analyze EEG data via an intuitive web interface. YASA Flaskified also supports straightforward deployment on your own physical or virtual server using a preconfigured deployment script. 
```

remrama replied:
Agreed, a bulleted list is preferred. And actually I think that the bullet could be placed under the "Development" subheading. A section titled "Related Projects" might be overkill for just this, unless we plan to add other sleep analysis software there as well. If left as-is, I would probably add some other packages to the list later.

raphaelvallat commented on docs/faq.rst:
```diff
@@ -219,6 +219,39 @@ Others
+      <div id="related_projects" class="panel-collapse collapse">
+        <div class="panel-body">
+
+**YASA Flaskified**
```
LGTM but I'd love to hear from @remrama's before final approval.

remrama replied:
- The docs just got changed to restructured text (instead of HTML). The formatting is much simpler, but I'm not sure how to handle this merge (since #194 is already merged into `master`). If anyone needs help with this, let me know.
- Did an extra `pip install --upgrade yasa` get added to Line 222? If so, that should be removed.
- I think the dropdown heading could be more informative than **Are there any related projects?** What about **Can I use YASA online?** or **Can YASA be deployed on a remote server?** If that makes sense...

raphaelvallat reviewed (commented):
Thanks @bartromb ! Looks good, I just had one suggestion on the README.rst file.

remrama commented on docs/faq.rst:
```diff
@@ -219,6 +219,39 @@ Others
+- **Deployment Options:** Users can deploy YASA Flaskified on their own physical or virtual server using a preconfigured deployment script. This allows for private, secure, and scalable use of the application.
+- **Integration with YASA:** The application is built around the YASA library, ensuring full compatibility and access to its core features, including sleep staging, event detection, and more.
+
+**Where to find it?**
```
- In general this section could have lots of text reduced. Try to avoid headings/questions _within_ FAQ headings. For example, instead of "Where to find it?" and then answering, just say "YASA Flaskified can be found at [insert link]", or even better, just make a statement about YASA Flaskified with a hyperlink.
- But we don't need a long description -- people should be going to the YASA Flaskified site for all the details. This helps long-term maintenance, so if something changes about YASA Flaskified, we don't have to update things here as well.
- I would suggest to be clearer that this is requires deployment of a server. I know it is said here, but I would _lead_ with that in the first sentence somehow. While this tool might be user-friendly for developers, I think many casual users might be misled into thinking they can just drag their files into the tool and run analyses. Including words like "server" and "deploy" will be enough to drop the hint.

bartromb replied:
"Related Projects
~~~~~~~~~~~~~~~~~

YASA Flaskified is a web-based tool built on YASA that allows EEG analysis through a browser interface. It requires deployment on a server, which can be done using the preconfigured scripts available in the repository. For more details, visit the **[YASA Flaskified GitHub repository](https://github.com/bartromb/YASAFlaskified)**"

bartromb commented:
Hi @raphaelvallat,

I’ve made the requested changes to the pull request:

    Reduced the text in the YASA Flaskified sections for clarity.
    Removed questions/headings in the FAQ and replaced them with concise statements.
    Emphasized that YASA Flaskified requires server deployment, making this clear in the first sentence.

You can view the updated changes [here](https://github.com/raphaelvallat/yasa/pull/198/files).

Let me know if there’s anything else you’d like me to adjust!

Best regards,
@bartromb

remrama reviewed (approved):
Hi @bartromb, thanks for making another adjustment. I like it! I think this should be merged after one last clarification:

I would only switch the markdown hyperlinks back to restructured text format. They were right in the first commit (using backticks `` `this <format>`_``), but they are markdown in the latest version (`[this](format)`). As far as I know, the markdown won't render in the `.rst` file, but let me know if I'm wrong about this.

bartromb commented:
You are completely right.  I’ll switch them back to the proper reStructuredText format as they were in the first commit. Thank you for catching that!

I’ll make this adjustment right away.

Best regards,
@bartromb

remrama commented:
@raphaelvallat do you have any other comments here? If not I can handle the merging of this. I'll check the docs online right after to make sure the FAQ and all that is still formatting properly.

bartromb reviewed (commented):
Looks good, thanks

raphaelvallat commented:
> do you have any other comments here?

All good for me, thank you!
